src/csharp/Microsoft.Spark.Extensions.Delta.UnitTest/Microsoft.Spark.Extensions.Delta.UnitTest.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
```

Comment:
are these needed ?

Reply:
Removed

Reply:
Actually, it seems that without this thing, the test runner can't find the tests.  So I will keep it.